ambry-network/src/main/java/com.github.ambry.network/SSLTransmission.java

```diff
@@ -138,7 +137,7 @@ public void close() {
       socketChannel.close();
     } catch (IOException ie) {
       metrics.selectorCloseSocketErrorCount.inc();
-      logger.warn("Failed to send SSL close message ", ie);
```

Comment:
?

Reply:
This log message ends up being pretty noisy in practice. Often SSLTransmission.close() is called when a machine is down and the connection failed, which means that there is no way to actually send the ssl close message.